config/uberon.yml

```diff
@@ -20,18 +20,43 @@ entries:
 
 - exact: /phenoscape-ext.owl
   replacement: http://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/edit/phenoscape-ext.owl
+  
+- exact: /composite-vertebrate.owl
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-vertebrate.owl
+  
+- exact: /composite-vertebrate.obo
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-vertebrate.owl
+
+- exact: /composite-metazoan.owl
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-metazoan.owl
+  
+- exact: /composite-metazoan.obo
+  replacement: https://github.com/obophenotype/uberon/releases/latest/download/composite-metazoan.obo
 
 - exact: /experimental/ceph.owl
   replacement: http://purl.obolibrary.org/obo/ceph.owl
 
 - prefix: /references/
-  replacement: http://svn.code.sf.net/p/obo/svn/uberon/trunk/reference/
+  replacement: https://raw.githubusercontent.com/obophenotype/uberon/master/reference/
 
 - prefix: /homology/
   replacement: https://phenoscape.svn.sourceforge.net/svnroot/phenoscape/trunk/vocab/homology/
 
+# Previously Uberon was releaseon on svn.code.sf.net. The following redirect essentially redirects any release pre-2020 (2019, 2018 etc) to that svn.
+- prefix: /releases/201
+  replacement: http://svn.code.sf.net/p/obo/svn/uberon/releases/201
+
```

Comment:
I'd feel better if there were a few tests for this, like:

```
  tests:
    - from: /releases/2014-03-26b/
      to: http://svn.code.sf.net/p/obo/svn/uberon/releases/2014-03-26b/
```

Reply:
I didnt even know that tests is a thing! Nice one.

Reply:
Ok I added some tests.